src/components/FooterComponent.vue

```diff
@@ -19,6 +19,33 @@
         Please cite <a href="https://www.ncbi.nlm.nih.gov/pubmed/30165396">Van der Velde et al (2018)</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/21210979">Swertz et al (2010)</a> or <a href="https://www.ncbi.nlm.nih.gov/pubmed/17297480">Swertz &amp; Jansen(2007)</a> on use.
       </p>
     </div>
+    <div v-if="googleAnalyticsTrackingId || googleAnalyticsTrackingIdMolgenis">
```

Comment:
Suggest to move to separate component, use vue component instead of script in template, add test , and update readme.